global_vars/noninteractive/amazon-site.yml

```diff
@@ -0,0 +1,68 @@
+---
+# Example site specific configuration for a noninteractive AWS deployment.
+#
+# Copy this and edit it as needed before running streisand-new-cloud-server.
+#
+
+streisand_noninteractive: true
+confirmation: true
+
+# The SSH private key that Ansible will use to connect to the Streisand node.
+#
+# The corresponding public key must be added to the Digital Ocean control panel
+# and the name given to it referenced below in the do_ssh_name variable.
+# The corresponding public key must be uploaded to Digital Ocean and the name
+# given to it referenced below in the do_ssh_name variable.
+streisand_ssh_private_key: "~/.ssh/id_rsa"
+
+vpn_clients: 5
+
+streisand_l2tp_enabled: yes
+streisand_openconnect_enabled: yes
+streisand_openvpn_enabled: yes
+streisand_shadowsocks_enabled: yes
+streisand_ssh_forward_enabled: yes
+streisand_stunnel_enabled: yes
+streisand_tinyproxy_enabled: yes
+streisand_tor_enabled: yes
+streisand_wireguard_enabled: yes
+
+# The AWS region number.
+# 1. Asia Pacific   (Mumbai)
+# 2. Asia Pacific   (Seoul)
+# 3. Asia Pacific   (Singapore)
+# 4. Asia Pacific   (Sydney)
+# 5. Asia Pacific   (Tokyo)
+# 6. Canada         (Central)
+# 7. EU             (Frankfurt)
+# 8. EU             (Ireland)
+# 9. EU             (London)
+# 10. South America (Sao Paulo)
+# 11. US East       (Northern Virginia)
+# 12. US East       (Ohio)
+# 13. US West       (Northern California)
+# 14. US West       (Oregon)
+#
+# Note that this must be a string representation of a number, not a number.
```

Comment:
Perhaps clearer expressed as 'Note: aws_region_var must be a number in quotes, e.g. "3" not 3.'

Reply:
The other defaults files should probably have their `region_var` comments updated for clarity as well.